server/actions/Square/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { Client, Environment } from "square";
+import { preProcessFile } from "typescript";
```

Comment:
I think this may have been auto-imported by VS Code somehow. It doesn't look like it's used anywhere in the file, so I think it's safe to remove this.